cvmfs/CMakeLists.txt

```diff
@@ -425,6 +470,37 @@ if (BUILD_SERVER)
   endif (BUILD_SERVER_DEBUG)
 endif (BUILD_SERVER)
 
+if (BUILD_PRELOADER)
+  add_executable (cvmfs_preload ${CVMFS_PRELOADER_SOURCES})
+
+  if (LIBCURL_BUILTIN)
+    add_dependencies (cvmfs_preload libcares libcurl)
+    set (PRELOADER_LIBS ${CARES_ARCHIVE} ${PRELOADER_LIBS} )
+    set (PRELOADER_LIBS ${LIBCURL_ARCHIVE} ${PRELOADER_LIBS} )
+  endif (LIBCURL_BUILTIN)
+
+  if (PACPARSER_BUILTIN)
+    add_dependencies (cvmfs_preload libpacparser)
+    set (PRELOADER_LIBS ${PACPARSER_ARCHIVE} ${PRELOADER_LIBS} )
+  endif (PACPARSER_BUILTIN)
+
+  if (SQLITE3_BUILTIN)
+    add_dependencies (cvmfs_preload sqlite3)
+    set (PRELOADER_LIBS ${SQLITE3_ARCHIVE}  ${PRELOADER_LIBS} )
+  endif (SQLITE3_BUILTIN)
+
+  if (ZLIB_BUILTIN)
+    add_dependencies (cvmfs_preload zlib)
+    set (PRELOADER_LIBS ${ZLIB_ARCHIVE} ${PRELOADER_LIBS} )
+  endif (ZLIB_BUILTIN)
+
+  if (SPARSEHASH_BUILTIN)
+    add_dependencies (cvmfs_preload sparsehash)
+  endif (SPARSEHASH_BUILTIN)
+
```

Comment:
I think there are some dependencies missing: libsha2, vjson, libtbb?  `${TBB_DEBUG_LIBRARIES}` should be `${TBB_LIBRARIES}`, no?